src/providers/VoiceFocusProvider/index.tsx

```diff
@@ -90,15 +90,18 @@ const VoiceFocusProvider: React.FC<Props> = ({
     logger.info(
       `Add Amazon Voice Focus to the following audio input device ${device}`
     );
-
-    if (voiceFocusDevice) {
-      const vf = await voiceFocusDevice.chooseNewInnerDevice(device);
-      logger.info(
-        'Re-used the same internal state to create an Amazon Voice Focus transform device.'
-      );
-      setVoiceFocusDevice(vf);
-      return vf;
-    }
+    // TODO: We don't need to intialize a new transformer every time we create a voice focus transformer device
```

Comment:
Delete unused code. (Or keep the code with your TODO comment.)

Reply:
I think we should just keep the TODO comment, its an improvement we can add later